org-roam.el

```diff
@@ -694,18 +694,25 @@ Valid states are 'visible, 'exists and 'none."
        ((< (window-height) h)
         (enlarge-window (- h (window-height))))))))
 
-(defun org-roam--setup-buffer ()
-  "Setup the `org-roam' buffer at the `org-roam-buffer-position'."
-  (let ((window (get-buffer-window)))
+(defun org-roam--set-up-buffer ()
+  "Set up the `org-roam' buffer at the `org-roam-buffer-position'."
+  (let ((window (get-buffer-window))
+        (position
+         (if (member org-roam-buffer-position '(right left top bottom))
+             org-roam-buffer-position
+           (let ((text-quoting-style 'grave))
+             (message "Invalid org-roam-buffer-position: %s. Defaulting to 'right"
```

Comment:
Maybe use `display-warning` here.

Reply:
Would you prefer `display-warning` or `lwarn`? With display warning we would have to pass the string into `format` to embed the invalid value, `lwarn` allows format specifiers in the warning message string.

Reply:
`lwarn` works too!